src/components/Editor/custom-hooks/useMarkerRef.ts

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from 'react';
```

Comment:
we use camelCase convention in project so `custom-hooks` directory is not a valid name.